packages/spor-elm/src/Spor/InfoTag.elm

```diff
@@ -0,0 +1,166 @@
+module Spor.InfoTag exposing (..)
+
+{-| A component for displaying info tags
+
+
+## Config
+
+@docs init, withVariant, withSize, withTitle, withDescription, withColor
+
+
+## Display
+
+@docs toHtml
+
+-}
+
+import Css exposing (Color)
+import Html.Styled as Html exposing (Html)
+import Html.Styled.Attributes as Attributes
+import Spor.Common.Types exposing (Size(..), Variant(..))
+import Spor.LineTagIcon as LineTagIcon
+import Spor.LineTagText as LineTagText
+import Spor.Token.Size.Spacing as Spacing
+
+
+{-| A component for displaying info tags
+-}
+type InfoTag
+    = InfoTag Options
+
+
+type alias Options =
+    { variant : Variant
+    , size : Size
+    , title : String
+    , description : Maybe String
+    , color : Maybe Color
+    }
+
+
+
+-- CONFIG
+
+
+{-| Create an initial configuration for a `InfoTag` component.
+-}
+init : InfoTag
+init =
+    InfoTag
+        { variant = LocalTrain
+        , size = Sm
```

Comment:
Jeg har gått for "Md" som default i React-implementasjonen - vi burde om ikke annet legge oss på samme størrelse som default.